### PR TITLE
[SPARK-21572][Spark Shell] Add description on how to exit the spark-shell in the welcome message

### PR DESCRIPTION
**Pull request by davidxdh**
When the user uses the spark-shell, does not know how to exit. So we need to add description on how to exit in the welcome message.
[https://issues.apache.org/jira/browse/SPARK-21572](https://issues.apache.org/jira/browse/SPARK-21572)

### CODE REVIEW

File: repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala [hunk 1]
```diff
@@ -84,6 +84,7 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
     echo(welcomeMsg)
     echo("Type in expressions to have them evaluated.")
     echo("Type :help for more information.")
+    echo("Type ctrl + c to exit.")
```
Comment (srowen) [inline repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala]: Ctrl-D or `:quit` exits. Scala shell doesn't print this and I think ctrl-D is widely understood to exit processes like this. Ctrl-C is an unnecessarily hard exit but also works. Is it really unclear to anyone?

Comment (davidxdh) [inline repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala]: Thanks for your advice. I do not think everyone knows the way to exit the scala, especially for beginner, and the clear explanation is helpful to them.

**Comment by jiangxb1987** [comment]
I’m not convinced this would be useful, could you close this please?

**Comment by HyukjinKwon** [comment]
I also don't think it is worth being printed.

**Comment by davidxdh** [comment]
OK. I close this PR.

**Pull request closed without merging**